spec/unit/java.spec.js

```diff
@@ -76,6 +76,19 @@ describe('Java', () => {
             Java.__set__('javaIsEnsured', false);
         });
 
+        it('CORDOVA_JAVA_HOME overrides JAVA_HOME', async () => {
+            spyOn(utils, 'forgivingWhichSync').and.returnValue('');
+
+            const env = {
+                CORDOVA_JAVA_HOME: '/tmp/jdk'
+            };
+
+            await Java._ensure(env);
+
+            expect(env.PATH.split(path.delimiter))
+                .toContain(path.join(env.JAVA_HOME, 'bin'));
```

Comment:
This is maybe nitpicking. (Feel free to ignore or comment it, off course )
But I expect the test to be without env.JAVA_HOME . 
I expect it more like
```
expect(env.PATH.split(path.delimiter)).toContain(path.join('/tmp/jdk', 'bin'));
```
You know exactly what it should be. Assert the exact value.
Now, when someone changes the business logic around JAVA_HOME and doesn't use the CORDOVA_JAVA_HOME in the correct way, the test could still succeed.

Reply:
Nope, I think I understand what you mean, this test could be improved. Let me know what you think about my change. I added an explicit assertion for `JAVA_HOME` as we want to ensure `JAVA_HOME` is set for our process (so that sub processes can be set appropriately), and changed it to be hard-coded `/tmp/jdk` as that is our chosen test value.